addons/sale_project/views/project_task_views.xml

```diff
@@ -49,6 +49,28 @@
         <field name="model">project.project</field>
         <field name="inherit_id" ref="project.edit_project"/>
         <field name="arch" type="xml">
+            <div name="button_box" position="inside">
+                <button class="oe_stat_button" type="object" name="action_view_sos" icon="fa-dollar" attrs="{'invisible': ['|', '|', ('allow_billable', '=', False), ('partner_id', '=', False), ('sale_order_count', '=', 0)]}" groups="sales_team.group_sale_salesman" context="{'create_for_project': active_id, 'default_project_id': active_id, 'default_partner_id': partner_id}">
```

Comment:
```suggestion
                <button
                    class="oe_stat_button"
                    type="object"
                    name="action_view_sos"
                    icon="fa-dollar"
                    attrs="{
                        'invisible': [
                            '|',
                               '|',
                                   ('allow_billable', '=', False),
                                   ('partner_id', '=', False),
                               ('sale_order_count', '=', 0),
                        ],
                    }"
                    groups="sales_team.group_sale_salesman"
                    context="{
                        'create_for_project': active_id,
                        'default_project_id': active_id,
                        'default_partner_id': partner_id,
                    }"
                >
```
Is it too much? :joy:
Xavier and I are unsure. Looks like in general, people want to have as little line as possible in xml files, but why?
I think it is way more readable like this, right?

Reply:
Sometimes, too many lines can hurt readability too, but here, I think it could be helpful :slightly_smiling_face: